webpack/dev.js

```diff
@@ -1,25 +1,25 @@
-const { merge } = require("webpack-merge");
-const common = require("./common");
-const webpack = require("webpack");
-const path = require("path");
-const MiniCssExtractPlugin = require("mini-css-extract-plugin");
-const HtmlWebpackPlugin = require("html-webpack-plugin");
-const InterpolateHtmlPlugin = require("react-dev-utils/InterpolateHtmlPlugin");
-const paths = require("../config/paths");
-const getClientEnvironment = require("../config/env");
+const { merge } = require('webpack-merge');
+const common = require('./common');
+const webpack = require('webpack');
+const path = require('path');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const InterpolateHtmlPlugin = require('react-dev-utils/InterpolateHtmlPlugin');
+const paths = require('../config/paths');
+const getClientEnvironment = require('../config/env');
 
-const publicUrl = "";
+const publicUrl = '';
 const env = getClientEnvironment(publicUrl);
 
 module.exports = merge(common, {
-  mode: "development",
-  devtool: "inline-source-map",
+  mode: 'development',
+  devtool: 'inline-source-map',
   devServer: {
     compress: true,
     open: true,
     port: 7000,
     client: {
-      overlay: true,
+      overlay: false,
```

Comment:
lets keep the overlay. Will make sure we don't miss important warnings and errors

Reply:
will revert it.

Reply:
reverted